Gemfile.lock

```diff
@@ -126,7 +126,7 @@ GEM
     ffaker (2.11.0)
     ffi (1.9.25)
     formatador (0.2.5)
-    fuubar (2.4.0)
```

Comment:
this is already in master, please rebase

Reply:
I will, I fixed it before it was fixed in master to make sure that the tests can pass.

Reply:
Done.